src/views/product/Product.tsx

```diff
@@ -12,7 +12,7 @@ import { cartSelector } from "../../store/selectors";
 
 type Props = {
   productVariant: FetchVariantResponse | null;
-  // categories: Category[];
+  // categories: Category[]
```

Comment:
Is there any reason why these properties are commented out? Will the actual code serve some purpose later in the future? And if so could you put a comment that explains this.